reflection.md

```diff
@@ -2,8 +2,12 @@
 
 ## Which operators were easiest to implement and why?
 
+The binary operators for union and concat were easiest to implement. Since normal math operations (addition/subtraction) are binary operators in many programming languages, the concept is easier to grasp. These operators are also syntactic sugar operators, which means there are also less logic to implement.
```

Comment:
I agree that these operators were the easiest, especially because we've already worked with them before.